chapter03/src/main/java/com/github/sebin/chapter03/Account.java

```diff
@@ -0,0 +1,87 @@
+package com.github.sebin.chapter03;
+
+public class Account {
+	private int accountID;
+	private User customer;
+	private String accountDescription;
+	private double accountBalance;
+	private double accountMaxBalance;
+	
+	
+	/**
+	 * @param accountID
+	 * @param customer
+	 * @param accountDescription
+	 * @param accountBalance
+	 * @param accountMaxBalance
+	 */
+	public Account(final int accountID,
+			final User customer,
+			final String accountDescription,
+			final double accountBalance,
+			final double accountMaxBalance) {
```

Comment:
I am not sure, it is a right thing to do. Someone convinced me few years back parameters should be final, so it will tell the reader or the code, it is never going to change(and that should be the normal behavior).

- String you can assign a new value if it is not final.
- primitive you can change inside the method, but it won't reflect in the caller. But i don't want any confusion :)